.travis.yml

```diff
@@ -1,20 +1,37 @@
 language: node_js
+
 sudo: required
+
+before_cache:
+  - rm -f ./node_modules/.bin/which
+
+cache:
+  - directories:
+      - "node_modules"
+      - "vendor"
+      - "_site/assets/processed-img"
+
 script:
-  - ./_scripts/run-tests.sh
+  - node_modules/eslint/bin/eslint.js _comments-app/
```

Comment:
@kostajh Why include these steps here instead of in the `run-tests.sh` script?

Reply:
Good question... I was thinking that since `script` takes an array of values, we may as well use that. But now that I think about it more, it's better to use `./_scripts/run-tests.sh` since we use the same script locally. My fault! Let me know if you want me to push a commit reverting that bit.